client/src/components/ProfileAvatar/ProfileAvatar.tsx

```diff
@@ -0,0 +1,15 @@
+import Avatar from '@material-ui/core/Avatar';
+import useStyles from './useStyles';
+import { User } from '../../interface/User';
+
+interface Props {
+  user: User;
+}
+
+const ProfileAvatar = ({ user }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return <Avatar alt="Profile Image" src={`https://robohash.org/${user.email}.png`} className={classes.avatar} />;
```

Comment:
Hai @DejanKatovic, Have a small question for you, is the Profile Avatar Component actually necessary?, since there is only a single component inside and not any additional logic .

Reply:
I thought like you, but, in in previous code, I saw AvatarDisplay with only a single component and that's why I made that component. Let me know if you want me to combine it.

Reply:
Hi @DejanKatovic, I second Ajay's comment above, we don't need a separate component for `ProfileAvatar` (You can leave it as of now, just keep it mind for future reference).